### ISSUE TITLE
error in Mildom

### ISSUE THREAD

**Issue by NoodlesStamps** [tf0101/VideoChatGet#51]
Thanks for a great program!
i have one question,
Does Mildom scraping work now?
I get this error

> ruby 2.7.0p0 (2019-12-25 revision 647ee6f091) [aarch64-linux-gnu]

>  videochatget https://www.mildom.com/playback/10006741/
identifier:mildom videourl:https://www.mildom.com/playback/10006741/
Traceback (most recent call last):
        6: from /usr/local/bin/videochatget:23:in `<main>'
        5: from /usr/local/bin/videochatget:23:in `load'
        4: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/exe/videochatget:5:in `<top (required)>'
        3: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/video_chat_get.rb:36:in `url_route'
        2: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/video_chat_get.rb:36:in `new'
        1: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/site_list/mildom_analyze.rb:21:in `initialize'
/var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/site_list/mildom_analyze.rb:28:in `videoid_get': undefined method `split' for nil:NilClass (NoMethodError)

No slash

>  videochatget https://www.mildom.com/playback/10438247
identifier:mildom videourl:https://www.mildom.com/playback/10438247
Traceback (most recent call last):
        6: from /usr/local/bin/videochatget:23:in `<main>'
        5: from /usr/local/bin/videochatget:23:in `load'
        4: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/exe/videochatget:5:in `<top (required)>'
        3: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/video_chat_get.rb:36:in `url_route'
        2: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/video_chat_get.rb:36:in `new'
        1: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/site_list/mildom_analyze.rb:21:in `initialize'
/var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/site_list/mildom_analyze.rb:28:in `videoid_get': undefined method `split' for nil:NilClass (NoMethodError)

With sudo 

>  sudo videochatget https://www.mildom.com/playback/10438247
identifier:mildom videourl:https://www.mildom.com/playback/10438247
Traceback (most recent call last):
        6: from /usr/local/bin/videochatget:23:in `<main>'
        5: from /usr/local/bin/videochatget:23:in `load'
        4: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/exe/videochatget:5:in `<top (required)>'
        3: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/video_chat_get.rb:36:in `url_route'
        2: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/video_chat_get.rb:36:in `new'
        1: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/site_list/mildom_analyze.rb:21:in `initialize'
/var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/site_list/mildom_analyze.rb:28:in `videoid_get': undefined method `split' for nil:NilClass (NoMethodError)

Other sites for instance Youtube , i have been sucessful.

**Comment by tf0101** [comment]
The format of the specified URL is different.
This package allows you to retrieve archived chats for delivery, but not chats that are being broadcast in real time.

For example in mildom.
Accept URLs of the following format.
>https://www.mildom.com/playback/10438247/video_id

You need the video_id in this way.

**Comment by NoodlesStamps** [comment]
Thanks for the reply!
Example : https://www.mildom.com/playback/10438247/10438247-burrbdrl5h2ve7kra8r0
is VideoID  "10438247-burrbdrl5h2ve7kra8r0"?
Or is this part? ”burrbdrl5h2ve7kra8r0”

>  videochatget https://www.mildom.com/playback/10438247/10438247-burrbdrl5h2ve7kra8r0
identifier:mildom videourl:https://www.mildom.com/playback/10438247/10438247-burrbdrl5h2ve7kra8r0
Traceback (most recent call last):
        6: from /usr/local/bin/videochatget:23:in `<main>'
        5: from /usr/local/bin/videochatget:23:in `load'
        4: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/exe/videochatget:5:in `<top (required)>'
        3: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/video_chat_get.rb:36:in `url_route'
        2: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/video_chat_get.rb:36:in `new'
        1: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/site_list/mildom_analyze.rb:21:in `initialize'
/var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/site_list/mildom_analyze.rb:28:in `videoid_get': undefined method `split' for nil:NilClass (NoMethodError)

> videochatget https://www.mildom.com/playback/10438247/burrbdrl5h2ve7kra8r0
identifier:mildom videourl:https://www.mildom.com/playback/10438247/burrbdrl5h2ve7kra8r0
Traceback (most recent call last):
        6: from /usr/local/bin/videochatget:23:in `<main>'
        5: from /usr/local/bin/videochatget:23:in `load'
        4: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/exe/videochatget:5:in `<top (required)>'
        3: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/video_chat_get.rb:36:in `url_route'
        2: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/video_chat_get.rb:36:in `new'
        1: from /var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/site_list/mildom_analyze.rb:21:in `initialize'
/var/lib/gems/2.7.0/gems/video_chat_get-0.1.5/lib/site_list/mildom_analyze.rb:28:in `videoid_get': undefined method `split' for nil:NilClass (NoMethodError)

Sorry if I am fatally mistaken

**Comment by tf0101** [comment]
There was an error in the package, so we've fixed it and updated the version.
Your problem will be solved by updating to video_chat_get (0.1.6).

Update to video_chat_get (v0.1.6) by following these steps
```
$ gem update video_chat_get
```
or
```
$ bundle update video_chat_get
```

The video_id is in the following format
VideoID 10438247-burrbdrl5h2ve7kra8r0


````
$ videochatget https://www.mildom.com/playback/10438247/10438247-burrbdrl5h2ve7kra8r0
````
This seems to solve the problem.

**Comment by NoodlesStamps** [comment]
thank you for your prompt response
It has worked fine! 
Thanks!
